tests/js/server/aql/aql-profiler.js

```diff
@@ -81,40 +83,84 @@ function ahuacatlProfilerTestSuite () {
     UpsertBlock, ScatterBlock, DistributeBlock, IResearchViewUnorderedBlock,
     IResearchViewBlock, IResearchViewOrderedBlock } = profHelper;
 
-  // See the limit tests (e.g. testLimitBlock3) for limit() and skip().
+  // See the limit tests (e.g. testLimitBlock3) for limit() and offset().
   const additionalLimitTestRowCounts = [
     // limit() = 1000 ± 1:
     1332, 1333, 1334,
-    // skip() = 1000 ± 1:
+    // offset() = 1000 ± 1:
```

Comment:
```suggestion
    // offset() = 4000 ± 1:
```

Reply:
?

Reply:
What's meant is, that for `rows` in `[3999, 4000, 4003, 4004]`, `offset(rows) = 1000 ± 1`.

Reply:
The idea is to choose `rows` in a way, that `offset(rows)` is just around the size of a batch, so we catch those corner cases.